api/src/main/java/team/unnamed/creative/model/ItemTransform.java

```diff
@@ -246,6 +243,18 @@ public Builder scale(Vector3Float scale) {
             return this;
         }
 
+        public Vector3Float getRotation() {
```

Comment:
Please be consistent with the rest of the code, we don't prefix getters with `get`, in this case it would just be `rotation()`